src/payment/strategies/paypal-commerce/paypal-commerce-request-sender.ts

```diff
@@ -27,6 +32,10 @@ export default class PaypalCommerceRequestSender {
             provider = 'paypalcommercecredit';
         }
 
+        if (this._configObject) {
+            provider = this._configObject.provider;
```

Comment:
Is there a reason why you can't just pass `params` to include `isAPM` flag? That way is consistent with the rest i.e.: `paypalcommercecredit`

Reply:
fixed